src/wallet/spend.cpp

```diff
@@ -492,6 +499,7 @@ std::optional<SelectionResult> SelectCoins(const CWallet& wallet, const std::vec
          */
         preset_inputs.Insert(coin, 0, false, 0, 0, false);
     }
+    // TODO: value_to_select -= preset_inputs.GetSelectionAmount();
```

Comment:
In 6423998409736be5728bb5826cfa89e1033ed816 "WIP. Return change from SelectionResult"

This is currently unnecessary because in the loop above, the values of preset inputs are subtracted from `value_to_select`.